ExadelTimeTrackingSystem/ExadelTimeTrackingSystem.BusinessLogic/Services/Abstract/IProjectService.cs

```diff
@@ -16,5 +16,7 @@ public interface IProjectService
         Task<List<string>> GetNamesAsync();
 
         Task<List<string>> GetActivitiesAsync(Guid id);
+
+        Task<string> GetProjectNameAsync(Guid id);
```

Comment:
GetNameAsync - it's already clear that name of project, cause it's in ProjectService